src/containers/Accounts/AMM/AMMAccounts/AMMAccountHeader/test/AMMAccountHeader.test.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import { I18nextProvider } from 'react-i18next'
+import thunk from 'redux-thunk'
+import { Provider } from 'react-redux'
+import { MemoryRouter } from 'react-router'
+import { initialState } from 'rootReducer'
+import i18n from 'i18nTestConfig'
+import { AMMAccountHeader, AmmDataType } from '../AMMAccountHeader'
+
+const configureMockStore = require('redux-mock-store').default
```

Comment:
Why is there a `require` used here? It looks like `configureMockStore` is usually directly imported from `'redux-mock-store'`

Reply:
This is a remnant of a altered code. Removing it now.